### ISSUE TITLE
How to run this for single image

### ISSUE THREAD

**Issue by santhoshkolloju** [paarandika/Gradient-Approx-to-improve-OCR#1]
How can use the pertained preprocessor and test on a single image?

**Comment by paarandika** [comment]
You can use the code in eval_prep.py to test on a single image. Currently, there is no script in here to run the trained model on any image other than the datasets given. I will add such a script soon.

**Comment by NeilduToit13** [comment]
Hi Paarandika I'm just adding my vote here for some guidance on running this over our own images. I'd really like to be able to test tesseract on my raw images and on my images after being preprocessed by your models and see whether I obtain any improvement. I've looked over the source for some time but I'm not able to easily understand where the entry points are. A script to run this on a single image, or on a directory with our own images, as well as pointers for retraining/finetuning the models if that's needed, would be very much appreciated and allow us to make use of your work and provide feedback. Thank you.

**Comment by ejkitchen** [comment]
I would like to echo the comments above. Would really like to be able to run this easily on custom images. Also for the fine-tuning and retraining, would be nice if you had a quick guide.
